package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nodenamo",
-  "version": "2.0.1",
+  "version": "2.1.0",
```

Comment:
I am doing: 

- Break change for a major version. 
- New feature for a minor version.
- Bug fixes for a patch.

Since this is a bug fix, can you do 2.0.2?

Reply:
Updated